test/CMakeLists.txt

```diff
@@ -927,6 +953,17 @@ ecbuild_add_test( TARGET  test_${PROJECT_NAME}_airnow
                           -o testrun/airnow_2020081306.nc"
                           airnow_2020081306.nc ${IODA_CONV_COMP_TOL_ZERO})
 
+ecbuild_add_test( TARGET  test_iodaconv_generic_gnssro_bufr
```

Comment:
Why is this added?

Reply:
Was moved but probably to the wrong section (was in bufr section but is not part of that effort as it uses python for its processing. Dunno were it should go.

Reply:
Moved this test back to where it was before as I don't think it belongs in this section.